dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -81,6 +81,12 @@ public GHRepository getRepo(String repoName) throws IOException {
         if (image.substring(image.lastIndexOf(' ') + 1).length() <= 1) {
             throw new IOException("Invalid image name.");
         }
+
+        // replace dots with spaces in image name, for github search to work
+        if(image.contains(".")) {
```

Comment:
This is a very broad change that has some pretty bad consequences. The worst consequence is that this can expand the search space which will trigger #169 

You might want to take a look at how we handled this for other portions of the images in #212. It'd also be possible to gate this functionality behind a new command-line flag but I still think it'd be best to incorporate a similar strategy to #212

Reply:
Hi @justinharringa , we were wondering if you could create code similar to #212 for this. The changes in this PR are just to show how we are resolving the issue on our end for now by creating a new image locally. Thanks!

Reply:
No worries. 😄  I would like to get some time to do this but work has been quite busy lately and finding time in my free time has also been a bit of a challenge. If you'd like to take a shot at it, that'd be awesome. But also appreciate you helping illustrate what has worked well for you. 😄